packages/ember-test-app/tests/integration/components/mktg/footer-test.ts

```diff
@@ -0,0 +1,72 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | components | mktg/footer', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(hbs`<Mktg::Footer />`);
+
+    assert.dom('footer').exists();
+  });
+
+  test('it contains a horizontal line', async function (assert) {
+    await render(hbs`<Mktg::Footer @hasHorizontalLine={{true}}/>`);
+
+    assert.dom('footer div hr').exists();
+  });
+
+  test('it contains a navigation bar', async function (assert) {
+    await render(hbs`<Mktg::Footer>
+      <:nav>
+        Nav Test
+      </:nav>
+    </Mktg::Footer>`);
```

Comment:
For writing multiline templates, there's a different format we're using:
```suggestion
    await render(hbs`
      <Mktg::Footer>
        <:nav>
          Nav Test
        </:nav>
      </Mktg::Footer>
    `);
```